worldedit-core/src/main/java/com/fastasyncworldedit/core/extension/factory/parser/mask/AdjacentMaskParser.java

```diff
@@ -43,7 +45,23 @@ protected Mask parseFromInput(@Nonnull String[] arguments, ParserContext context
             max = min;
         }
         if (max >= 8 && min == 1) {
-            return new AdjacentAnyMask(subMask);
+            int miny;
+            int maxy;
```

Comment:
Camel casing? Afaik, minY, maxY is what's been used in more spots.

Reply:
Agreed. 🐫 Casing please.